build/linux/debian/dep-lists.js

```diff
@@ -0,0 +1,168 @@
+"use strict";
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+Object.defineProperty(exports, "__esModule", { value: true });
+exports.referenceGeneratedDepsByArch = exports.bundledDeps = exports.recommendedDeps = exports.additionalDeps = void 0;
+// Based on https://source.chromium.org/chromium/chromium/src/+/main:chrome/installer/linux/debian/additional_deps
+// Additional dependencies not in the dpkg-shlibdeps output.
+exports.additionalDeps = [
+    'ca-certificates',
+    'fonts-liberation',
+    'libgtk-3-0 (>= 3.9.10) | libgtk-4-1',
+    'libnss3 (>= 3.26)',
+    'wget',
+    'libcurl3-gnutls | libcurl3-nss | libcurl4 | libcurl3',
+    'xdg-utils (>= 1.0.2)' // OS integration
+];
+// Based on https://source.chromium.org/chromium/chromium/src/+/main:chrome/installer/linux/debian/manual_recommends
+// Dependencies that we can only recommend
+// for now since some of the older distros don't support them.
+exports.recommendedDeps = [
+    'libu2f-udev',
+    'libvulkan1' // Move to additionalDeps once support for Trusty and Jessie are dropped.
+];
+// Based on https://source.chromium.org/chromium/chromium/src/+/refs/tags/98.0.4758.109:chrome/installer/linux/BUILD.gn;l=64-80
+// and the Linux Archive build
+// Shared library dependencies that we already bundle.
+exports.bundledDeps = [
+    'libEGL.so',
+    'libGLESv2.so',
+    'libvulkan.so.1',
+    'swiftshader_libEGL.so',
+    'swiftshader_libGLESv2.so',
+    'libvk_swiftshader.so',
+    'libffmpeg.so'
+];
+exports.referenceGeneratedDepsByArch = {
+    'amd64': [
+        'ca-certificates',
+        'fonts-liberation',
+        'libasound2 (>= 1.0.16)',
+        'libatk-bridge2.0-0 (>= 2.5.3)',
+        'libatk1.0-0 (>= 2.2.0)',
+        'libatspi2.0-0 (>= 2.9.90)',
+        'libc6 (>= 2.14)',
+        'libc6 (>= 2.17)',
+        'libc6 (>= 2.2.5)',
+        'libcairo2 (>= 1.6.0)',
+        'libcurl3-gnutls | libcurl3-nss | libcurl4 | libcurl3',
+        'libdbus-1-3 (>= 1.5.12)',
+        'libdrm2 (>= 2.4.38)',
+        'libexpat1 (>= 2.0.1)',
+        'libgbm1 (>= 17.1.0~rc2)',
+        'libgcc1 (>= 1:3.0)',
+        'libgdk-pixbuf-2.0-0 (>= 2.22.0)',
```

Comment:
This seems to be a new dependency. On my Xubuntu 20.04 machine, I get a missing dependency error while using dpkg to install the package.

Reply:
Same issue on Ubuntu 16.04.

Reply:
I confirmed these dependencies are from the code-insiders binary.